prov/verbs/src/ep_rdm/verbs_rdm_cm.c

```diff
@@ -75,7 +75,7 @@ static inline ssize_t
 fi_ibv_rdm_batch_repost_receives(struct fi_ibv_rdm_conn *conn,
 				 struct fi_ibv_rdm_ep *ep, int num_to_post)
 {
-	const size_t idx = (conn->cm_role == FI_VERBS_CM_SELF) ? 1 : 0;
+	const size_t idx = (conn->cm_role == FI_VERBS_CM_SELF);
```

Comment:
I don't think this is a good idea... AFAIR according to C standard (before C99) boolean operation returns `0` and `not 0`, i mean no guarantee that result of `!0` will be equal to `1`. Below, it's used as index of array.

Reply:
Yes, you're right.
But our code isn't compatible with previous C standard (I mean before C99).

To keep this more clear and safe, I'll return back. Thanks!